kdumpctl

```diff
@@ -1729,8 +1792,12 @@ main()
 			reset_crashkernel_for_installed_kernel "$2"
 		fi
 		;;
+	test)
+		shift
+		kdump_test "$@"
+		;;
 	*)
-		dinfo $"Usage: $0 {estimate|start|stop|status|restart|reload|rebuild|reset-crashkernel|propagate|showmem}"
+		dinfo $"Usage: $0 {estimate|start|stop|status|restart|reload|rebuild|reset-crashkernel|propagate|showmem|test [--force]}"
```

Comment:
I would drop the `--force` here as the other sub-commands don't specify their options here as well.
That should be done in `kdumpctl.8` where you should add a section for the new command anyway ;-)

Reply:
Thanks for the suggestion, agreed, will update the man entry as well.